.github/labeler.yml

```diff
@@ -0,0 +1,9 @@
+📖 documentation: 
```

Comment:
@quinnj this line is the only one that needs to change unless you have a "📖 documentation" label

Reply:
oh you do, nvm!